### PR TITLE
(SEO) Add description and keywords for troubleshooting related pages

### PR DESCRIPTION
**Pull request by IsAmrish**
Signed-off-by: isamrish <askmaurya48@gmail.com>

This PR will add two new fields in every page related to troubleshooting.
The fields are
- description
- keywords

This will help in improving the SEO for the troubleshooting pages

### CODE REVIEW

**Review by satyapriyamishra222 — APPROVED** [review]

**Review by rakeshPRaghu — APPROVED** [review]

File: docs/versioned_docs/version-2.11.0/troubleshooting/cstor.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ---
 id: cstor
 title: Troubleshooting OpenEBS - cStor
+keywords:
+  - cStor
+  - cStor troubleshooting
```
Comment (satyapriyamishra222) [inline docs/versioned_docs/version-2.11.0/troubleshooting/cstor.md:6]: Add 
-OpenEBS
- OpenEBS community

File: docs/versioned_docs/version-2.11.0/troubleshooting/install.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 ---
 id: install
 title: Troubleshooting OpenEBS Install
+keywords:
+  - OpenEBS installation
```
Comment (satyapriyamishra222) [inline docs/versioned_docs/version-2.11.0/troubleshooting/install.md:5]: ++
OpenEBS

File: docs/versioned_docs/version-2.11.0/troubleshooting/localpv.md [hunk 3]
```diff
@@ -1,6 +1,12 @@
 ---
 id: localpv
 title: Troubleshooting OpenEBS - Dynamic LocalPV
+keywords:
+  - LocalPV
```
Comment (satyapriyamishra222) [inline docs/versioned_docs/version-2.11.0/troubleshooting/localpv.md:5]: ++
OpenEBS

File: docs/versioned_docs/version-2.11.0/troubleshooting/mayastor.md [hunk 4]
```diff
@@ -1,6 +1,10 @@
 ---
 id: mayastor
 title: Troubleshooting Mayastor
+keywords:
```
Comment (satyapriyamishra222) [inline docs/versioned_docs/version-2.11.0/troubleshooting/mayastor.md:4]: ++
OpenEBS

File: docs/versioned_docs/version-2.11.0/troubleshooting/cstor.md (hunk 1 above)
Comment (Pallavi-PH) [reply docs/versioned_docs/version-2.11.0/troubleshooting/cstor.md:8]: SEO keywords are the words and phrases that searchers enter into search engines and these help them land on the correct page. IMO, `OpenEBS community` will be the right keyword for only the community page.

**Review by Pallavi-PH — APPROVED** [review]

**Review by satyapriyamishra222 — APPROVED** [review]

**Pull request merged**